networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java

```diff
@@ -69,6 +77,13 @@ public BlobsSidecarsByRangeMessageHandler(
   @Override
   public Optional<RpcException> validateRequest(
       final String protocolId, final BlobsSidecarsByRangeRequestMessage request) {
+    if (request.getCount().isGreaterThan(maxRequestSize)) {
```

Comment:
Not sure about this. We need to make sure we do not respond with more than `maxRequestSize` (yeah the name is misleading) but we can accept a bigger range, like what we do in `BeaconBlocksByRangeMessageHandler`. Reading the specs seems to be aligned with the blocks by range behaviour.

Reply:
agree, removed this validation